HTTP2-Core.package/SqueakSSL.extension/instance/alpnProtos.st

```diff
@@ -0,0 +1,3 @@
+*HTTP2-Core
+alpnProtos
+	^ self primitiveSSL: handle getStringProperty: 3.
```

Comment:
Konstanten bitte auf Klassenseite auslagern

Reply:
Wird in allen anderen {get,set}{Int,String}Property-Aufrufen auch so gehandhabt. Wir wären mit SqueakSSL-Standards inkonsistent, wenn wir das in eine Konstante auslagern würden.